src/Symfony/Component/VarDumper/Dumper/CliDumper.php

```diff
@@ -538,7 +534,7 @@ protected function supportsColors()
     /**
      * {@inheritdoc}
      */
-    protected function dumpLine($depth, $endOfValue = false)
```

Comment:
we should keep the 2nd argument, to document that the method is called with it at least

Reply:
The parameter is never read and looked like legacy to me. I'd rather adjust the single call that uses the parameter. But anyway, my change was out of scope, so I'll revert it for now.